cognee/shared/utils.py

```
@@ -443,15 +450,19 @@
     )
     p.add_tools(hover_tool)
 
+    # Get the latest Unix timestamp as an integer
+    timestamp = int(time.time())
+
+    # Construct your filename
+    filename = f"{timestamp}.png"
```

Ruff (0.8.2)

Local variable `filename` is assigned to but never used

Remove assignment to unused variable `filename`

(F841)